open_academy/views/session.xml

```diff
@@ -0,0 +1,57 @@
+<odoo>
+    <data>
+      <record model="ir.ui.view" id="session_view_form">
+        <field name="name">session.view.form</field>
+        <field name="model">session</field>
+        <field name="arch" type="xml">
+          <form>
+            <sheet>
+              <group>
+                <field name="name"/>
+              </group>
+              <notebook>
+                <page string="Start date">
+                  <field name="start_date"/>
+                </page>
+                <page string="Duration">
+                    <field name="duration" widget='timesheet_uom'/>
+                  </page>
+                  <page string="Number of seats">
+                    <field name="number_seat"/>
+                  </page>
+              </notebook>
+            </sheet>
+          </form>
+        </field>
+      </record>
+      <record model="ir.ui.view" id="session_view_tree">
+        <field name="name">session.view.tree</field>
+        <field name="model">session</field>
+        <field name="arch" type="xml">
+          <tree>
+            <field name="name"/>
+            <field name="start_date"/>
+            <field name="duration" widget='timesheet_uom'/>
+            <field name="number_seat"/>
+          </tree>
+        </field>
+      </record>
+      <record model="ir.ui.view" id="session_view_search">
+        <field name="name">session.view.search</field>
+        <field name="model">session</field>
+        <field name="arch" type="xml">
+          <search>
+            <field name='name'/>
+            <field name="start_date"/>
+            <field name="duration" widget='timesheet_uom'/>
+            <field name="number_seat"/>
+          </search>
+        </field>
+      </record>
+      <record id='session_action_display_view' model='ir.actions.act_window'>
+        <field name='name'>Session</field>
+        <field name='res_model'>session</field>
+        <field name='view_mode'>tree,form,search</field>
```

Comment:
Please address this comment c:

Is it necessary add "search" to this field?

Reply:
When I did not add it, the app did not work for me. So, I added and everything worked fine.

Reply:
Ok, it was just a doubt.

Reply:
Sorry @Jerudiel, I still don't know enough to answer your question, I'm a little behind in the training document, but Francisco surely could answer that in a few minutes! :D

Reply:
@rolandojduartem remove search from view_mode